### ISSUE TITLE
WT fails to start after Windows login

### ISSUE THREAD

**Issue by angelog0** [microsoft/terminal#1954]
Even with this preview (v0.2.1831.0) I see the following.

Install WT from the Store.
Start it: all is OK. Close it.
Shutdown the PC (switch it off)
Start the PC (switch it on)
Log in to Windows
Start WT: an empty, transparent window shows up for a while then in closes: WT fails to start. Tried to start, to start, ... but does not work...

Now to start WT I have to start first MinTTY (MSYS2 terminal) or my MSYS2/MINGW64 build of Emacs from master. (Tried with Chromium, M. Store, T.Bird etc. but did not work, maybe there are other apps with wich this would work...)

Are there registry keys it writes? which?

Thanks.

**Comment by DHowett-MSFT** [comment]
Interesting. Do you happen to have a Windows Error Reporting event in your event log from when it fails to launch? We'd love to look for related crashes.

**Comment by angelog0** [comment]
@DHowett-MSFT, sorry for the late replay but waited for 0.3 release... I had uninstalled 0.2..

I installed 0.3 from the Store but not started it after installation. I rebooted, logged in in Windows and than started WT. It started. So closed it and rebooted another time repeating these steps: but now, it didn't start, just the empty (transparent) window (better: its contour rectangle).

With right mouse on Start I selected `Event Viewer`  and searched under `Windows registers` (`Windows logs` could be another translation) and under `Application` but not found errors. Notice that I started WT at 15:14 but the logs where for 15:13 and 15:15, nothing for 15:14

May you suggest where to search? eventually from command line..

Thanks,
  Angelo.

...obviously, after closing `Event Viewer`, retrying,  WT started.

**Comment by dgweller** [comment]
Similar, but not same.  Under no scenario does wt successfully run on my machine.  

Are there any command line params I can apply when manually launching wt.exe?

**Comment by angelog0** [comment]
The only feedbak I have is that also 0.4 fails in the same manner. I don't find any `core dump`or similar or messages of the failure. Maybe you have to design your application so that some log of the failure is released. Don't ask to use `Feedback Hub` because I don't like it and it is between bloat applications I remove after Windows installation... let me know if there is a way to start WT from command line, sometimes one can catch some output there... or simply close this: I see other users have similar failure.

BTW, some user commented the the issue could be related to `powershell`, but here PS works and WT starts if I start it after some other applications. Just re-read the topics.

**Comment by angelog0** [comment]
May be I am wrong but in the last release (`0.5+`) the issue has gone..  Thanks.

Some comments/observations.

1. Opacity/transparency. When `useAcrylic` is enable (for example for `CMD` profile) the behavior is : when the window has focus it is transparent otherwise it is opaque. I think it would be better the contrary: opaque when it has focus, transparent in the other case. Or, at least, it should have the appropriate setting.

2. When in Ubuntu one uses `less` to read a text file, the mouse wheel does not work to scroll it: only up/down arrow keys work. Many terminals have that.

3. Usually I use MinTTY from MSYS2 on Windows so I have the habit to copy/paste with the mouse double clicking and pasting with the mouse wheel. Doing that wrongly in WT did not work, obviously, but after a while I tried it caused the crash of WT (no messages etc., just disappeared... may be I had enabled `copyOnSelect`). Many terminal work that way.. Why not WT?

4. Sometimes the tab width it is too short. Consider the CMD tab. It contains the icon, the string `cmd` and the `x` to close it but often just selecting this tab results in closing it because one click too near the `x`... May be the tabs should have a larger width even if the text it contains is short.

5. Last but not least, why you do not introduce an option to set the position of its top-left corner so that when one launch it, the window is always in the same position (something like the initial `Cols`, `Rows`...)

**Comment by DHowett-MSFT** [comment]
1. #1375 acrylic is "flipped" by system policy, there's a heated debate on it going on right now
2. #376 #545
3. #2807 #2927 
3. #597 and you can set the tab title with the `tabTitle` profile option
5. #1043

**Comment by DHowett-MSFT** [comment]
I'm surprised it started working for you. Unrelatedly, we identified an issue where _it looks like on some machines the console driver just isn't automatically loaded_, which speaks to a potential issue in our pseudoconsole init path.

**Comment by zadjii-msft** [comment]
Hey @angelog0, a couple things:

* First, please try and keep each issue thread specific to the issue being discussed in that issue. This particular thread is dealing with the bug of the Terminal displaying an empty window on launch.
* Second, please search before filing new issues. As Dustin pointed out, there are issues already for each of those bullet points.
* If you do come across an issue that's not already being tracked, please try and keep each issue to a singular problem. A big grab-bag of feedback is much harder to deal with than 5 individual issues for separate pieces of feedback.

**Comment by angelog0** [comment]
@DHowett-MSFT, I installed this new version from the store, then rebooted. Login and launch WT first time. OK. rebooted. Then many times: Login and launch WT. It started. Add MSYS2, MSYS2 MinGW32/64; change the font (first DejaVU Sans Mono installed `ad hoc` then Cascadia Code from WT), change the cursor to `fillBox` and its color for Ubuntu, MSYS2 & C., change the Ubuntu icon to `ubuntu.png`... It always start in the procedure described above!

since the last time I had installed WT there was a new W10 monthly update and I decided to uninstall LibreOffice installed from the `msi` and to install only the portable version (unpacking with `7z` the `paf.exe`, more or less a manual installation)

The only issue I have now with WT: it crashes copy/pasting, too often but, at the moment, I have not a recipe to reproduce it systematically.

@zadjii-msft, sorry for the noise.
